Feature listingsby categoryand sub category

varuntheocean opened the pull request:
1.Getlistingsbycategoryandsubcategory
2.GetlistingsbyEmail
3.Test case for GetlistingsbyEmail
4.Test case for Getlistingsbycategoryandsubcategory

varuntheocean committed:
1.GetListingsByCategoryAndSubCategory
2.GetListingsByEmail
3.Test Case for GetListingsByCategoryAndSubCategory
4.Test Case for GetListingsByEmail

varuntheocean committed:
Code changes on Repository testcase for listing by category and sub category

santoshkale3 reviewed (commented):
You need to filter out those records submitted by user.
Add one more parameter as email and filter result where submittedby is not equal to email.

varuntheocean committed:
1.Submittedby is not equal to Email
2. Status feature Added fo GetListingByCategory,GetListingBySubCategory and GetListingByCategoryandSubCategory.Retuns only Active records.
3.Pagination added to GetListingByCategoryandSubCategory

varuntheocean committed:
code changes
